src/mailer.py

```diff
@@ -0,0 +1,55 @@
+import boto3
+from botocore.exceptions import ClientError
+
+
+CHARSET = "UTF-8"
+
+
+class EmailReport():
+    def __init__(self, sender):
+        self.client = boto3.client('ses')
+        self.sender = sender
+
+    def send(self, logger, recipients, cc, subject, body):
+        bodyhtml = """<html>
+        <head></head>
+        <body>
+        <p>{}</p>
+        </body>
+        </html>
+        """.format(body.replace("\n", '<br/>'))
+
+        try:
+            #Provide the contents of the email.
+            response = self.client.send_email(
+                Destination={
+                    'ToAddresses': recipients,
+                    'CcAddresses': cc
+                },
+                Message={
+                    'Body': {
+                        'Html': {
+                            'Charset': CHARSET,
+                            'Data': bodyhtml,
+                        },
+                        'Text': {
+                            'Charset': CHARSET,
+                            'Data': body,
+                        },
+                    },
+                    'Subject': {
+                        'Charset': CHARSET,
+                        'Data': subject,
+                    },
+                },
+                Source=self.sender,
+                # If you are not using a configuration set, comment or delete the
+                # following line
+                # ConfigurationSetName=CONFIGURATION_SET,
+            )
+        # Display an error if something goes wrong.
+        except ClientError as e:
+            logger.debug(e.response['Error']['Message'])
```

Comment:
Since this is a system error, it probably indicates an error in the email config. I think we should add an exit code here so that the lambda function itself fails if we get a client error, which will trigger a CloudWatch alarm if one exists.

Maybe like this:

```
except ClientError as e:
  logger.debug(e.response['Error']['Message'])
  raise e
```

Reply:
Thanks! Updated to fail lambda on client error.